src/configs/pm_model_config.yaml

```diff
@@ -0,0 +1,9 @@
+encoder: squezeenet
+num_classes: 9
+prototype_shape: [9, 512, 1, 1]
+top_k: 1
+epsilon: 1e-4
+num_warm_epochs: 0
+push_start: 0
```

Comment:
@mak-en move to [configs](https://github.com/quantori/edema-quantification/tree/model_draft/configs)